multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -121,6 +123,15 @@ def forward(
         data = batch[self.image_key]
         data["img_metas"] = [img_metas.data[0] for img_metas in data["img_metas"]]
         data["img"] = [img.data[0] for img in data["img"]]
+
+        device = next(self.model.parameters()).device  # model device
```

Comment:
Will this interfere with pytorch-lightening? Also, does this work with DDP on multi-cards or multi-nodes?

Reply:
I guess it should work with ddp. Each gpu runs this code and gets its device id.